.schema/config.schema.json

```diff
@@ -445,6 +445,14 @@
             "POST"
           ]
         },
+        "proxy_headers": {
```

Comment:
Let's call this `forwarded_http_headers` :)

Reply:
Yep. That's a much clearer name for the variable :)